### ISSUE TITLE
`$ wandbox help` is not work

### ISSUE THREAD

**Issue by Fuyutsubaki** [srz-zumix/wandbox-api#3]
https://github.com/srz-zumix/wandbox-api/blob/787612e5bba4ce0bb3e5d7296a262a7df25faceb/wandbox/cli.py#L138

`args.command` is not exist

**Comment by srz-zumix** [comment]
fixed: #5 


```
$ wandbox help
usage:  wandbox help [-h] subcommand{list,compiler,lang,option,permlink,run}
wandbox help: error: the following arguments are required: subcommand
```

```
$ wandbox help permlink
usage: wandbox permlink [-h] id

get permlink

positional arguments:
  id          permlink id

optional arguments:
  -h, --help  show this help message and exit
```

**Comment by srz-zumix** [comment]
Thank you for reporting!
I released version 0.9.4 which fixes the problem.
